### PR TITLE
 Auth - adapt docker setup and move to .code TLD 

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

**Comment by blazarecki** [comment]
You can change the target branch to `auth2`.

**Comment by B-Galati** [comment]
@blazarecki done sorry for that

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -1,12 +1,12 @@
 parameters:
-    env(DATABASE_HOST):         db
+    env(DATABASE_HOST):         enmarche_db
```
Comment (blazarecki) [inline app/config/parameters.yml.dist]: Now, there is no need to prefix container with `enmarche_` you can revert this in all files.

File: docker/dev/supervisord.conf [hunk 2]
```diff
@@ -11,10 +12,21 @@ stderr_logfile=/dev/stderr
...
 autostart=true
 autorestart=true
 stdout_logfile=/dev/stdout
 stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
+
+[program:worker-api-broadcast-user]
+priority=2
+command=/usr/bin/php bin/console rabbitmq:consumer -w api_broadcast_user
```
Comment (blazarecki) [inline docker/dev/supervisord.conf]: You can comment this since it's not import yep.

**Comment by B-Galati** [comment]
@blazarecki done

File: docker-compose.yml [hunk 3]
```diff
@@ -1,24 +1,33 @@
...
-            - /app/var/cache
-            - /app/var/sessions
 
     db:
         image: mysql:5.7
         environment:
             MYSQL_ROOT_PASSWORD: root
 
-    redis:
+    enmarche_redis:
```
Comment (blazarecki) [inline docker-compose.yml]: Same here, please use redis instead of enmarche_redis.

**Comment by B-Galati** [comment]
@blazarecki Done thanks !

**Pull request merged**